docker-compose.yml

```diff
@@ -123,7 +123,7 @@ services:
       RESULT_BACKEND: redis://redis:6379
       CELERY_QUEUE: "registry_queue"
       SQLALCHEMY_DATABASE_URI: "postgresql+psycopg2://user:password@postgres-registry:5432/default_db"
-      LOCALHOST_PROXY_PORTS: ":5005  ${EXTRA_PROXY_PORTS}"
+      LOCALHOST_PROXY_PORTS: ":5005 :9090 ${EXTRA_PROXY_PORTS}"
```

Comment:
We should always have the same port forwarding for the api and the worker container. It can happen, that some requests are done inline in the api process in the future.